projects/fsastorefrontlib/src/cms-components/my-account/quote/quotes/quotes.component.ts

```diff
@@ -217,7 +226,7 @@ export class QuotesComponent implements OnInit, OnDestroy {
   }
 
   goToDetailsPage(quote: InsuranceQuote) {
-    if (this.quotesByCategory[quote.defaultCategory.name]?.length >= 2) {
+    if (this.quotesByCategory[quote.defaultCategory.name]?.length > 1) {
```

Comment:
This condition has more sense because only when is more than one quote from the same category on the Quotes&Aplication page, the Compare button should be visible on the Quote details page.